zbus/src/connection/socket/mod.rs

```diff
@@ -312,14 +312,14 @@ pub trait WriteHalf: std::fmt::Debug + Send + Sync + 'static {
     /// After this call, it is valid for all reading and writing operations to fail.
     async fn close(&mut self) -> io::Result<()>;
 
-    /// Supports passing file descriptors.
+    /// Return whether passing file descriptors is supported.
```

Comment:
This `Return` is also unneeded and inconsistent with the other getters here.

Reply:
Done